appveyor.yml

```diff
@@ -0,0 +1,32 @@
+# version format
+version: 1.0.{build}-{branch}
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
```

Comment:
I don't quite understand this one - does this mean that we don't build PRs at all?

Also, note that I have not read the security policy on AppVeyor but I know that Travis CI only builds PRs from the members (to prevent anyone from submitting a rogue PR that can expose sensitive information during the build).